index.js

```diff
@@ -6,22 +6,49 @@ const chalk = require('chalk');
 const prettyBytes = require('pretty-bytes');
 const StreamCounter = require('stream-counter');
 const gzipSize = require('gzip-size');
+const brotliSize = require('brotli-size');
 
 module.exports = options => {
 	options = {
 		pretty: true,
 		showTotal: true,
+		uncompressed: !options || options.uncompressed || !(options.gzip || options.brotli),
 		...options
 	};
 
-	let totalSize = 0;
 	let fileCount = 0;
+	const totalSize = {};
 
-	function log(what, size) {
+	function log(what, sizes) {
 		let {title} = options;
 		title = title ? chalk.cyan(title) + ' ' : '';
-		size = options.pretty ? prettyBytes(size) : (size + ' B');
-		fancyLog(title + what + ' ' + chalk.magenta(size) + (options.gzip ? chalk.gray(' (gzipped)') : ''));
+		const desc = {uncompressed: '', gzip: ' (gzipped)', brotli: (' (brotli)')};
+		const strings = Object.entries(sizes).map(([k, v]) => {
+			const size = options.pretty ? prettyBytes(v) : (v + ' B');
+			return chalk.magenta(size) + chalk.gray(desc[k]);
+		});
+
+		fancyLog(title + what + ' ' + strings.join(chalk.magenta(', ')));
+	}
+
+	function addPropWise(a, b) {
+		// eslint-disable-next-line guard-for-in
+		for (const k in b) {
```

Comment:
Use a `for-of` loop.

And use descriptive variable/function names. No abbreviations.